src/ssplit/ssplit.cpp

```diff
@@ -135,7 +135,8 @@ operator()(string_view* rest) const {
   // The chunker is the first step in sentence splitting.
   // It identifies candidate split points.
   static Regex Chunker_RE("\\s*" // whitespace
-                          "([^.?!]*)" // prefix: anything up to the potential EOS marker
+                          "[^.?!]*?" // non alphanumeric stuff
+                          "(\\p{Xan}*)" // alphanumeric prefix of the potential EOS marker
```

Comment:
```
(\\p{Xan})
1st Capturing Group (\\p{Xan})
\\ matches the character \ literally (case sensitive)
p{Xan matches the characters p{Xan literally (case sensitive)
} matches the character } literally (case sensitive)
* matches the previous token between zero and unlimited times, as many times as possible, giving back as needed (greedy)
```

Is this what is intended? Or is regex101 with PCRE2 doing something wrong?

Reply:
Can you add a respective test for this case and update samples or something to ensure this case now works as intended? Would be easier to approve.

Reply:
1. The double backslash is an escaped backslash. If post that directly in to a regex checker, you get the effect that you observe.
2.  Xan is a pcre2 extension https://www.pcre.org/current/doc/html/pcre2syntax.html#SEC6
3. A test case has been added to the test document.